grants/models.py

```diff
@@ -0,0 +1,55 @@
+# grants/models.py
+from django.db import models
+from django.utils import timezone
+
+
+class GrantApplication(models.Model):
+    GRANT_TYPES = (
+        ("TRAVEL", "Travel"),
+        ("TICKET", "Conference Ticket"),
+        ("FULL", "Full (Travel + Ticket)"),
+        ("SPEAKER", "Speaker"),
+    )
+
+    TICKET_CHOICES = (
+        ("YES", "Yes"),
+        ("NO", "No"),
+    )
+
+    timestamp = models.DateTimeField()
+    full_name = models.CharField(max_length=200)
+    email = models.EmailField(unique=True)
+    profession = models.CharField(max_length=200)
+    country_of_origin = models.CharField(max_length=100)
+    city_of_travel = models.CharField(max_length=100)
```

Comment:
IMHO, `profession`, `country_of_origin` and `city_of_travel` might be too much information to store.